### ISSUE TITLE
Remove `systemd` from `home-manager`?

### ISSUE THREAD

**Issue by drupol** [nix-community/home-manager#5274]
Hello,

Today I was looking for improving and reducing the size of Home Manager profile's closure.
During my investigation, I noticed that `systemd` is included by default, even in the smallest Home Manager profile.

With the smallest profile I could make, I can't go under 200M.

`systemd` on its own is 165M:

```
/nix/store/3np3qw5y5xarl4hxbhk9vj2d5kmgqsir-systemd-255.2               34.7M  165.3M
```

I use Home-Manager in a Docker instance where I only need to trigger the installation of the profile once and never use Home Manager later on. 

Therefore, I open this issue to discuss of the possibility to enable or disable `systemd`.

If it is already possible, please let me know.

**Comment by drupol** [comment]
To disable `systemd` with configuration:

```
  home.activation.reloadSystemd = lib.mkForce "";
```

**Comment by Samasaur1** [comment]
Out of curiosity, I tried to make a minimal home-manager config on Darwin, and I was able to get it down to 50.3M. And home-manager on Darwin definitely doesn't use systemd and doesn't seem to use launchd to activate configurations, so presumably this would be possible on Linux as well?

**Comment by drupol** [comment]
I opened a PR for that @ https://github.com/nix-community/home-manager/pull/5276

Feedback is very welcome!

**Comment by sumnerevans** [comment]
Closed by # #5276. Next time, please include "Closes #ISSUE_NUM" in the description text of the PR so that it automatically closes the corresponding issue.

**Comment by drupol** [comment]
You're right I'll double pay attention, thanks for doing it.
